ci/properties/go-publisher-ossf-slsa3.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "SLSA Go releaser",
+    "creator": "Open Source Security Foundation (OpenSSF)",
+    "description": "Compile your Golang project using a SLSA3 compliant builder.",
+    "iconName": "go-publisher-ossf-slsa3",
+    "categories": ["Continuous integration"]
```

Comment:
Add "Go" as well

Reply:
I'll defer to @josepalafox

I think we suggested using `SLSA verified builder` instead, since CI is a general term and this workflow is about secure build pipeline, namely SLSA

Wdut?

Reply:
Unfortuantely we do not have a category for "secure build" and hence we will use CI category. We will internally discuss this and plan for adding more categories but that will take time.